### PR TITLE
Vanilla

### PR DESCRIPTION
**Pull request by atharva-tendle**
Adds more architectures and multi-training.

### CODE REVIEW

**Comment by ayushm-agrawal** [comment]
Maybe a good idea to add a .gitignore and put pycache in there

File: data.py [hunk 1]
```diff
@@ -39,3 +50,100 @@ def load_cifar10(configs):
...
+    test_loader = torch.utils.data.DataLoader(
+        test_set, batch_size=configs.batch_size, num_workers=0)
+
+    print('Number of iterations required to get through training data of length {}: {}'.format(
+        len(train_set), len(train_loader)))
+
+    print(train_set.data.shape)
+    print(test_set.data.shape)
+
+    return {'train': train_loader, 'test': test_loader}
```
Comment (ayushm-agrawal) [inline data.py]: Two return statements?

File: utils.py [hunk 2]
```diff
@@ -1,50 +1,63 @@
...
+        elif training_type == "untrained":
+            print("Loading untrained Resnet18")
+            if configs.dataset == "MNIST" or configs.dataset == "FashionMNIST":
+                model = ResNet18(num_classes=10, input_channels=1)
+            elif configs.dataset == "CIFAR-100":
+                model = ResNet18(num_classes=100, input_channels=3)
+            else:
+                model = ResNet18(num_classes=10, input_channels=3)
+
+    elif model_name == "Xception":
```
Comment (ayushm-agrawal) [inline utils.py:33]: The indentation looks wrong here

File: vanilla.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class VanillaCNN2(nn.Module):
+    print("Loading VANILLA 2...")
+
+    def __init__(self):
```
Comment (ayushm-agrawal) [inline vanilla.py]: Need to pass channels and num classes here

File: xception.py [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+            'std': [0.5, 0.5, 0.5],
+            'num_classes': 1000,
+            'scale': 0.8975 # The resize parameter of the validation transform should be 333, and make sure to center crop at 299x299
+        }
+    }
+}
+
+def xception(num_classes=1000, pretrained='imagenet'):
+    
+    model = Xception(num_classes=num_classes)
```
Comment (ayushm-agrawal) [inline xception.py:190]: Should include input_channels

**Review by ayushm-agrawal — COMMENTED** [review]
Some comments and also would be a good idea to add a .gitignore and add __pycache__ inside it

File: utils.py (hunk 2 above)
Comment (atharva-tendle) [reply utils.py:33]: why?

File: xception.py (hunk 4 above)
Comment (atharva-tendle) [reply xception.py:190]: There was an issue with this that I wanted to discuss with you. We aren't doing pretrained runs today so we can discuss this.

**Pull request merged**